src/schema/v2/me/identity_verification.ts

```diff
@@ -0,0 +1,57 @@
+import {
+  GraphQLFieldConfig,
+  GraphQLObjectType,
+  GraphQLString,
+  GraphQLNonNull,
+} from "graphql"
+import { ResolverContext } from "types/graphql"
+import { InternalIDFields } from "schema/v1/object_identification"
+import dateField, { date } from "../fields/date"
+
+const dateFieldForVerificationExpiresAt: GraphQLFieldConfig<
+  any,
+  ResolverContext
+> = {
+  ...dateField,
+  resolve: (
+    { invitation_expires_at: expiresAt },
+    { format, timezone },
+    { defaultTimezone }
+  ) => {
+    const rawDate = expiresAt
+
+    const timezoneString = timezone ? timezone : defaultTimezone
+    return date(rawDate, format, timezoneString)
+  },
+}
+
+const IdentityVerificationType = new GraphQLObjectType<any, ResolverContext>({
+  name: "IdentityVerificationType",
+  fields: {
+    ...InternalIDFields,
+    state: {
+      type: new GraphQLNonNull(GraphQLString),
+      description: "Where the identity verification is in its lifecycle",
+    },
+    userID: {
+      type: new GraphQLNonNull(GraphQLString), // TODO: Ok to have an id string(?) here instead of the User object?
```

Comment:
Minor question here.

Reply:
It's up to you, and what the schema needs are. I would probably leave out entirely unless you actually need it.

Reply:
i _think_ its ok to have it as ID for now till there is a reason to resolve it with actual user. Knowing this is already under `me` i don't think we need that just yet.